core/templates/dev/head/services/contextual/UrlServiceSpec.ts

```diff
@@ -91,6 +93,11 @@ describe('Url Service', function() {
     expect(UrlService.getHash()).toBe(sampleHash);
   });
 
+  it('should correctly return the origin of window.location', function() {
+    var origin = 'http://sample.com';
```

Comment:
Inline this. There's already an "origin" declared outside this so it's a bit confusing to have two vars with the same name, but fortunately you don't need the interior var.

Reply:
Done